src/messages.js

```diff
@@ -181,7 +181,7 @@ var warnings = {
   W101: "Line is too long.",
   W102: null,
   W103: "The '{a}' property is deprecated.",
-  W104: "'{a}' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).",
+  W104: "'{a}' is available in ES6 (use 'esversion: 6') or Mozilla JS extensions (use moz).",
```

Comment:
We also need to decide what to do about the moz function.. in v3 will it be there? will it be removed? if it will be removed, then this PR is doing the right thing